tests/subscriptions/test_codecs.py

```diff
@@ -55,31 +61,45 @@ def test_decode(self):
         assert codec.decode(payload) == subscription
 
 
-class TestQueryResultCodec(BaseTest):
+class TestSubcriptionTaskResultCodec(BaseTest):
     def test_encode(self):
-        result = SubscriptionResult(
+        timestamp = datetime.now()
+
+        subscription_data = SubscriptionData(
+            1,
+            [],
+            [["count()", "", "count"]],
+            timedelta(minutes=1),
+            timedelta(minutes=1),
+        )
+
+        # XXX: This seems way too coupled to the dataset.
+        request = subscription_data.build_request(
+            get_dataset("events"), timestamp, None, Timer("timer")
+        )
```

Comment:
This is clunky but it's way easier than generating a `Request` object by hand right now.